webpack.config.js

```diff
@@ -25,8 +25,13 @@ module.exports = {
         loaders: [
             {
                 test: /\.less$/,
-                loader: ExtractTextPlugin.extract("style-loader", "css-loader!less-loader")
-            }
+                loader: ExtractTextPlugin.extract('style-loader', 'css-loader!less-loader')
+            },
+            {
+                test: /\.tpl\.html$/,
+                loader: 'raw-loader',
+                exclude: /node_modules/
+            },
```

Comment:
It tells webpack:
* If a file that ends in `.tpl.html` is required
* Use the raw-loader to process it (because it's not javascript)
* Except if the file/require is in `node_modules/` then ignore it

So basically every time you use `require('somePath/someTemplate.tpl.html')` in angular webpack will replace it with the template when it builds the app.

The one above was a little bit different because It has to run less so that we have a css file and then the extract plugin moves it to a separate file otherwise it would inline it with the javascript.